controllers/pipeline/pipeline_adapter_test.go

```diff
@@ -696,6 +699,40 @@ var _ = Describe("Pipeline Adapter", Ordered, func() {
 			Expect(found).To(BeFalse())
 		})
 
+		It("ensures build labels and annotations prefixed with 'build.appstudio' are propagated to the snapshot", func() {
+			snapshot, err := adapter.prepareSnapshotForPipelineRun(testpipelineRunBuild, hasComp, hasApp)
+			Expect(err).To(BeNil())
+			Expect(snapshot).ToNot(BeNil())
+
+			annotation, found := snapshot.GetAnnotations()["build.appstudio.openshift.io/repo"]
+			Expect(found).To(BeTrue())
+			Expect(annotation).To(Equal("https://github.com/devfile-samples/devfile-sample-go-basic?rev=c713067b0e65fb3de50d1f7c457eb51c2ab0dbb0"))
+
+			label, found := snapshot.GetLabels()["build.appstudio.redhat.com/target_branch"]
+			Expect(found).To(BeTrue())
+			Expect(label).To(Equal("main"))
+		})
+
+		It("ensures build labels and annotations non-prefixed with 'build.appstudio' are NOT propagated to the snapshot", func() {
+			snapshot, err := adapter.prepareSnapshotForPipelineRun(testpipelineRunBuild, hasComp, hasApp)
+			Expect(err).To(BeNil())
+			Expect(snapshot).ToNot(BeNil())
+
+			var found bool
+			// build annotations non-prefixed with 'build.appstudio' are not copied
+			_, found = testpipelineRunBuild.GetAnnotations()["appstudio.redhat.com/updateComponentOnSuccess"]
```

Comment:
```suggestion
			_, found := testpipelineRunBuild.GetAnnotations()["appstudio.redhat.com/updateComponentOnSuccess"]
```
Looks good to me, just a nit-pick: I think we can remove line 721 `var found bool` and use `_, found :=` here and then  replace `found1` with ` found` in next lines.

Reply:
Ack and fixed. Thank you